app/models/spree/shipment_decorator.rb

```diff
@@ -1,5 +1,12 @@
 Spree::Shipment.class_eval do
-  scope :exportable, -> { joins(:order).where('spree_shipments.state != ?', 'pending') }
+  scope :exportable, lambda {
```

Comment:
Can we refactor this to:

```ruby
query = joins(:order).merge(Spree::Order.complete) 
query.where.not(spree_shipments: { state: 'pending' }) if Spree::Config.require_payment_to_ship
```

Also I think it might make more sense to turn this into a class method instead of a scope? @mtomov what do you think?